yt_dlp/extractor/nrk.py

```diff
@@ -163,33 +171,36 @@ def call_playback_api(item, query=None):
         playable = manifest['playable']
 
         formats = []
+        subtitles = {}
         for asset in playable['assets']:
             if not isinstance(asset, dict):
                 continue
             if asset.get('encrypted'):
-                continue
+                pass  # Unencrypted stream no longer available
             format_url = url_or_none(asset.get('url'))
             if not format_url:
                 continue
             asset_format = (asset.get('format') or '').lower()
             if asset_format == 'hls' or determine_ext(format_url) == 'm3u8':
-                formats.extend(self._extract_nrk_formats(format_url, video_id))
+                fmts, subs = self._extract_nrk_formats_and_subtitles(format_url, video_id)
+                formats.extend(fmts)
+                self._merge_subtitles(subs, target=subtitles)
             elif asset_format == 'mp3':
                 formats.append({
                     'url': format_url,
                     'format_id': asset_format,
                     'vcodec': 'none',
                 })
 
-        data = call_playback_api('metadata')
+        data = self._call_api(try_get(manifest, lambda x: x['_links']['metadata']['href']), video_id, 'metadata')
 
-        preplay = data['preplay']
-        titles = preplay['titles']
-        title = titles['title']
-        alt_title = titles.get('subtitle')
+        preplay = try_get(data, lambda x: x['preplay'])
+        titles = try_get(preplay, lambda x: x['titles'])
+        title = try_get(titles, lambda x: x['title'])
+        alt_title = try_get(titles, lambda x: x['subtitle'])
```

Comment:
Okay, I thought maybe `try_get` was the preferred since I saw it used in the original code quite a bit in place of `.get()`. I'll start working on changing all instances of it to `traverse_obj` instead unless there's nothing to traverse, in which case I will use `object.get()` instead.